pegasus/sites.v3/code.org/public/index.haml

```diff
@@ -29,7 +29,6 @@ style_min: true
   - if request.language == "en"
     = view :curriculum_banner
     = view :homepage_below_hero_announcement
-    = view :professional_learning_marketing_banner, audience_text: 'you', link_address: '/educate/professional-learning/middle-high', button_label: 'Learn more', homepage: true
```

Comment:
Removing this superhero skinny banner because we'll have a superhero hero banner on the homepage.